src/components/ModalWithBackdrop.tsx

```diff
@@ -19,13 +19,13 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 import { Modal, ModalProps } from 'react-native'
 import styled from 'styled-components/native'
 
-interface ModalWithBackdropProps extends ModalProps {
+export interface ModalWithBackdropProps extends ModalProps {
   closeModal?: () => void
   color?: string
 }
 
 const ModalWithBackdrop = ({ children, closeModal, color, ...props }: ModalWithBackdropProps) => (
-  <Modal transparent={true} {...props} animationType="none">
+  <Modal transparent={true} animationType="none" {...props}>
```

Comment:
so that `animationType` can be overridden if needed.

Reply:
Yeah I forced it to none because these animations posed problems in the past. Let's just keep that in mind in case we see witness behaviours in the future :)

Reply:
It's good to default to none, but be able to override it if needed though, don't you agree?

Reply:
We are currently using `"fade"` in these places:
- Spinner modal
- Consolidation modal
- Color picker modal
- Verify Mnemonic modal
- Settings screen

We should check each individual case on iOS to make sure we don't get stuck anywhere.